src/main/java/backend/UpdateController.java

```diff
@@ -87,7 +89,13 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         return filterExprs.stream()
                 .filter(Qualifier::hasUpdatedQualifier)
                 .flatMap(filterExpr -> allModelIssues.stream()
-                        .filter(issue -> Qualifier.process(models, filterExpr, issue)))
+                        .filter(issue -> {
+                            try {
+                                return Qualifier.process(models, filterExpr, issue);
+                            } catch (SemanticException e) {
+                                return false;
+                            }
+                        }))
```

Comment:
Hmm, this is going to conflict with #1323... It might be best to wait until Martin is done. We can use a similar approach to carry the warnings to the UI.

I feel like we shouldn't try to distinguish warnings and errors at this point, since only one warning is returned (same as an error), and we will want a list of errors in future anyway. Can we just treat a missing assignee as an error, and change errors so that they all show in a tooltip (to use the new UI features implemented here, and because we want to move them out of the status bar in future anyway)? @damithc

We can work towards being able to collect multiple warnings/errors in future.